pkg/agent/openflow/pipeline.go

```diff
@@ -2159,10 +2163,15 @@ func (c *client) snatCommonFlows(nodeIP net.IP, localSubnet net.IPNet, localGate
 		// should bypass SNAT too. But it has been covered by the gatewayCT related flow generated in l3FwdFlowToGateway
 		// which forwards all reply traffic for such connections back to the gateway interface with the high priority.
 
-		// Send the traffic to external to SNATTable.
+		// This generates the flow to match the requests packets sourced from local Pods and destined for external, then
+		// forward the packets to SNATTable. Traffic cases include:
+		//	- Service request packets sourced from local Pods and destined for external Endpoint.
+		//  - Non-Service request packets sourced from local Pods and destined for external network.
```

Comment:
Since it includes both service request and non-service request, I think there is no point to list the two cases, and "request" is not the point of this flow, not "service". I think comments following "Traffic cases include" is not required.

Reply:
Updated.